### ISSUE TITLE
wazuh-api doesn't run in container

### ISSUE THREAD

**Issue by polachz** [wazuh/wazuh-docker#318]
Hallo.

I'm trying to run the latest wazuh cotainer (**docker.io/wazuh/wazuh:3.11.4_7.6.1**) on Centos8 by podman

Container is able to run, but api is not responsive. 

When I attach to the container and try to run api manually, I'm getting this:

>root@ed29dcc2ec12:/# /usr/bin/nodejs /var/ossec/api/app.js -f
>Dependencies not found. Try 'npm install' in /var/ossec/api. Exiting...

I have tried to update the api by apt-get update apt-get install wazuh-api, and also tried to reinstall the node.js. But without luck.

PS: I have indices that it is connected with mounted volume to the container:

> /srv/dddd/wazuh/wazuh/data:/var/ossec/data:Z

Because with container **docker.io/wazuh/wazuh** (seems that it's wazhuh 3.10)  I'm have different problem, - api writes that /var/ossec/logs/api.log is inacessible -> permission denied

Even i change this file to chmod 666, owner on root or ossec, still getting the error. 

When runs the container without this volume mounted, it works, but api is still corrupted:

root@3ac15b9376b6:/var/ossec/logs# cat api.log

> WazuhAPI 2020-03-22 14:14:20 : Listening on: https://:::55000
> WazuhAPI 2020-03-22 14:15:00 : [::ffff:172.17.17.15] GET /agents?offset=0&limit=1&q=id!%3D000 - 400 - error: '802'.
> WazuhAPI 2020-03-22 14:15:00 : [::ffff:172.17.17.15] GET /cluster/status? - 400 - error: '802'.
> WazuhAPI 2020-03-22 14:15:02 : [::ffff:172.17.17.15] GET /manager/info? - 400 - error: '802'.
> WazuhAPI 2020-03-22 14:15:02 : [::ffff:172.17.17.15] GET /manager/info? - 400 - error: '802'.
 
Going Out of mind how-to mage wazuh run in the container :(

Do you have an idea what's wrong?

**Comment by polachz** [comment]
OK, I have discovered the source of problems. 

The host location (partition) where I can mount the volume was mounted **nodev, noduid, noexec**

When i have mounted this only rw, all begin to work smoothly. 

As bonus, the Wazuh is now able to run as **rootless** container with podman, what's my target due security ;)

Then problem is solved. Maybe i only recommends to mention this fact and necessary rights in docker documentation.

Anyway, the issue has been solved and can be closed.
